extension/src/format/autoIndent.ts

```diff
@@ -5,13 +5,17 @@ import * as format from './listreader';
 import { LispAtom, Sexpression, indentationForNarrowStyle } from './sexpression';
 import { CursorPosition } from './listreader';
 
-class ElementRange {
+export class ElementRange {
     constructor() {
         this.startPos = null;
         this.endPos = null;
         this.quoted = false;
     }
 
+    getRange(document: vscode.TextDocument): vscode.Range{
+        return new vscode.Range(document.positionAt(this.startPos.offsetInDocument), document.positionAt(this.endPos.offsetInDocument));
```

Comment:
If I remember it correctly, the end of range should be the index of the next char that is right after the range.
See https://github.com/Autodesk-AutoCAD/AutoLispExt/blob/90a25dbe7e7282ff29346d5d22608070236db111/extension/src/format/autoIndent.ts#L39,
Please do some test to determine what to do.

Reply:
Thanks for the insight. I will look into this, it is entirely possible the ending parenthesis was missing and I just didn't notice.

Reply:
Still working on creating a proper test for this and the multi-line string bug. In both cases I need to re-install some of the stuff I shelved in a backup to do it and make sure none of that winds up in a commit.